### PR TITLE
SCAT-5118 supplier justification

### PR DESCRIPTION
**Pull request by sudhakar-reddy-konda**

### CODE REVIEW

File: cat/CaT-service.yaml [hunk 1]
```diff
@@ -1776,9 +1776,15 @@ paths:
...
-                items:
-                  $ref: 'https://raw.githubusercontent.com/Crown-Commercial-Service/ccs-api-definitions-common/main/CCS_OCDS_Standards/CCS-OCDS_Schema.yaml#/components/schemas/OrganizationReference'
+                type: object
+                title: ProcurementEventSuppliers
+                properties:
+                  suppliers:
+                    type: array
+                    items:
+                      $ref: 'https://raw.githubusercontent.com/Crown-Commercial-Service/ccs-api-definitions-common/main/CCS_OCDS_Standards/CCS-OCDS_Schema.yaml#/components/schemas/OrganizationReference'
+                  justificatiion:
```
Comment (tommyb82) [inline cat/CaT-service.yaml:1786]: typo - `justificatiion` --> `justification`

File: cat/CaT-service.yaml [hunk 2]
```diff
@@ -1776,9 +1776,15 @@ paths:
           content:
             application/json:
               schema:
-                type: array
-                items:
-                  $ref: 'https://raw.githubusercontent.com/Crown-Commercial-Service/ccs-api-definitions-common/main/CCS_OCDS_Standards/CCS-OCDS_Schema.yaml#/components/schemas/OrganizationReference'
+                type: object
```
Comment (tommyb82) [inline cat/CaT-service.yaml:1779]: It is usually worth extracting these objects into a dedicated schema type to coerce the OpenAPI generator into creating a dedicated class..

**Review by tommyb82 — CHANGES_REQUESTED** [review]
Just minor changes 👍

File: cat/CaT-service.yaml (hunk 1 above)
Comment (sudhakar-reddy-konda) [reply cat/CaT-service.yaml]: Thanks. Done

File: cat/CaT-service.yaml (hunk 2 above)
Comment (sudhakar-reddy-konda) [reply cat/CaT-service.yaml:1780]: Done. Please check

**Review by tommyb82 — APPROVED** [review]
Minor typo in description but all good

File: cat/CaT-service.yaml [hunk 3]
```diff
@@ -4100,6 +4096,17 @@ components:
             type: string
             nullable: true
             format: date-time
+    EventSuppliers:
+      description:
+        List of supplier belonfs to procurement event.
```
Comment (tommyb82) [inline cat/CaT-service.yaml:4101]: typo

**Pull request merged**